test/draw/gpu/pc/GpuSurfaceTest.ooc

```diff
@@ -139,7 +139,7 @@ GpuSurfaceTest: class extends Fixture {
 			gpuImage canvas drawLines(trianglePoints)
 			gpuImage canvas drawBox(FloatBox2D new(-lineLength, -lineLength, lineLength * 2.0f, lineLength * 2.0f))
 			//
-			// NOTE! The circle use a point size of 1.0f (OpenGLES3MapPoints in OpenGLES3Map.ooc)
+			// NOTE! The circle use a point size of 1.0f (OpenGLMapPoints in OpenGLMap.ooc)
```

Comment:
The plan is to be able to set the point size in the GPU API, this is not possible today. The point size is set to 1.0f however. I think the reason for this comment is to be clear that in this test a point size of 1.0f is expected, so when we can change it in the API we should set it to 1.0f in this test.